### PR TITLE
New term metastatic maglinant neoplasm in the brain

### LINKED ISSUE #7835: Request for new term "methicillin-susceptible staphylococcus aureus infectious disease"
### Label

methicillin-susceptible staphylococcus aureus infectious disease

### Synonyms

_No response_

### Synonym type

None

### Definition

Infections caused by a strain of Staphylococcus aureus that is susceptible to the action of the antibiotic methicillin.

### Parent term

MONDO:0005545

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

_No response_

### Additional comments

_No response_

### LINKED ISSUE #7839: Request for new term "Metastatic malignant neoplasm in the brain"
### Label

Metastatic malignant neoplasm in the brain

### Synonyms

_No response_

### Synonym type

None

### Definition

A metastatic malignant neoplasm affecting the brain.

### Parent term

MONDO:0001657 brain cancer or MONDO:00024880 metastatic malignant neoplasm

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

_No response_

### Additional comments

Some "metastatic malignant neoplasms" are defined as "secondary neoplasms", "metastatic malignant neoplasm in the lymph nodes" is defined as "neoplasm". Cancers currently only include "neoplasms", but not "secondary neoplasms" which means that some "metastatic malignant neoplasms" are not cancers, while "metastatic malignant neoplasm in the lymph nodes" is. The textual definitions for some cancers also explicitly mention metastases.

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7839 

- Definition
- Logical definition
- 2 Parents
- Term tracker
- 1 PMID

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -582755,6 +582755,18 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-23T20:59:58Z" xsd:
...
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7831" xsd:anyURI
 
+[Term]
+id: MONDO:1040026
+name: metastatic malignant neoplasm in the brain
+def: "Any human metastatic malignant neoplasm affecting the brain." [PMID:29262201]
+is_a: MONDO:0001657 {source="PMID:29262201"} ! brain cancer
+is_a: MONDO:0024880 {source="PMID:29262201"} ! metastatic malignant neoplasm
+intersection_of: MONDO:0024880 ! metastatic malignant neoplasm
+intersection_of: disease_arises_from_feature UBERON:0000955 ! brain
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:582765]: the relation should be "disease has location". 
Similar to 'metastatic malignant neoplasm in the eye' (MONDO:0044913) and 'metastatic malignant neoplasm in the spinal cord' (MONDO:0044912)

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@yshwetar 
The definition is great! :-) 
the logical definition should follow the [disease by location]( https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/location.yaml) pattern.  The relation you used in the logical definition should be updated.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (yshwetar) [reply src/ontology/mondo-edit.obo:582765]: Comment addressed

**Review by sabrinatoro — APPROVED** [review]
perfect. Approving and Merging

**Comment by sabrinatoro** [comment]
I resolved the conflict, I will merge after QC have passed.

**Pull request merged**